custom_components/waste_collection_schedule/waste_collection_schedule/source/lindau_ch.py

```diff
@@ -0,0 +1,61 @@
+import json
+from datetime import datetime
+from bs4 import BeautifulSoup
+
+import requests
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+
+TITLE = "Abfall Lindau"
+DESCRIPTION = "Source for Lindau waste collection."
+URL = "https://www.lindau.ch/abfalldaten"
+TEST_CASES = {
+    "Tagelswangen": {"city": "Tagelswangen", "types": "kehricht, grungut, papier und karton, altmetalle, hackseldienst"},
+    "Grafstal": {"city": "190", "types": "grungut, papier und karton"},
+}
+
+
+class Source:
+    def __init__(self, city, types):
```

Comment:
Remove types from source parameters. A source shall provide all waste types. Waste types can be filtered in the sensor configuration.

Reply:
Why does a source have to provide all types? This only makes the sensor config unnecessarily complicated!